### ISSUE TITLE
Темная тема не реализована.  Dark theme not implemented.

### ISSUE THREAD

**Issue by daniliammo** [Castro-Fidel/PortWINE#416]
Приложение не умеет использовать системную тему.

Никто больше не писал об этой проблеме.

![issue](https://github.com/user-attachments/assets/fccde9f9-5dfa-4494-8b74-9dfbd0df13e5)

**Comment by Htylol** [comment]
@daniliammo https://linux-gaming.ru/t/smena-svetloj-temy-fletpak-versii-na-tyomnuyu/564

**Comment by daniliammo** [comment]
> @daniliammo https://linux-gaming.ru/t/smena-svetloj-temy-fletpak-versii-na-tyomnuyu/564

У Меня не работает. Да это и не решение а костыль. Системная тема должна работать из коробки.

**Comment by Htylol** [comment]
@daniliammo https://github.com/Castro-Fidel/PortWINE/pull/420 добавил ПР в общем, если примут,  то в глобальных настройках можно тёмную тему будет выбрать

**Comment by Boria138** [comment]
> Системная тема должна работать из коробки.

Должна, но не работает и это проблема flatpak и порталов, а не PortProton сколько в интернете не ищи только такой костыль и предлагают

**Comment by daniliammo** [comment]
> > Системная тема должна работать из коробки.
> 
> Должна, но не работает и это проблема flatpak и порталов, а не PortProton сколько в интернете не ищи только такой костыль и предлагают

Не думаю что в порталах есть какая то проблема. В других флетпаках все работает. Похоже проблема в yad

**Comment by Boria138** [comment]
> > > Системная тема должна работать из коробки.
> > 
> > Должна, но не работает и это проблема flatpak и порталов, а не PortProton сколько в интернете не ищи только такой костыль и предлагают
> 
> Не думаю что в порталах есть какая то проблема. В других флетпаках все работает. Похоже проблема в yad

Нет, на гноме у меня тема была, в kde тоже видел у людей тему, так что это проблема SteamOS

**Comment by Boria138** [comment]
Ну и KDE тоже, я видел людей с темой в ней, но что они сделали так и не понял, у меня не вышло без костыля тему получить

**Comment by Boria138** [comment]
В общем в GNOME тема работает прям из коробки, ничего делать дополнительно не надо, а вот в KDE надо что-то сделать, но что я пока не выяснил, так что это проблема Flatpak, а не PortProton

**Comment by Boria138** [comment]
То что на скрине гном я если что вижу, я сужу из своего опыта, на арч тема была, на EndeavourOS тема была, на Ximper тема была, а что там в Fedora я не смотрел, может надо что-то доустановить или ещё что

**Comment by Castro-Fidel** [comment]
Выбор темы добавлен в глобальные настройки PP.
Если переключение работает, то закрывайте.

**Comment by daniliammo** [comment]
> Выбор темы добавлен в глобальные настройки PP. Если переключение работает, то закрывайте.

Автоматическое определение работать не будет?

**Comment by Htylol** [comment]
@daniliammo Как ты его предлагаешь реализовать во flatpak?
